lib/containers/artist.js

```diff
@@ -37,11 +37,19 @@ class Artist extends React.Component {
   render() {
     const windowDimensions = Dimensions.get('window');
     const commonPadding = windowDimensions.width > 700 ? 30 : 20;
+    const tabs = ['ABOUT'];
+    debugger;
+    if (this.props.artist.counts.artworks) {
+      tabs.push('WORKS');
+    }
+    if (this.props.artist.partner_shows.length) {
+      tabs.push('SHOWS');
```

Comment:
Could there be a case in which an artist has shows but not works? If so, I should change the logic [here](https://github.com/artsy/emission/pull/80/files#diff-f64004b75dbb599d0061fe4024381638L32). If not, it might be better to KISS for now.